### DISCUSSION TITLE
Subquery is not supported in the select list

### DISCUSSION THREAD

**Discussion by dewen2018** [apache/doris#10974] [A - General / Q&A] [answered]
select 
a.*
,(select sum(b.num) from b where a.id = b.rid) nums
from a;
这种子查询可以支持一下吗，现在会报这种错误
-----------------
Error Code: 1105. errCode = 2, detailMessage = Subquery is not supported in the select list.
-----------------

**Comment by hf200012** [comment, marked as answer]
这个目前不支持，后面会考虑支持

**Reply by fuhaooo** [reply]
现在支持了吗？

**Reply by YooLin** [reply]
现在支持了吗

**Reply by joeykoer** [reply]
2.1 还是不支持
